### ISSUE TITLE
Old Manchester CCGs showing on the map

### ISSUE THREAD

**Issue by richiecroker** [ebmdatalab/openprescribing#490]
The three old Manchester CCGs are still showing on the data map, with a list size and number of rx of 0.  There does not appear to be a single Manchester CCG boundary on the map (unless it is overlaid).

![image](https://user-images.githubusercontent.com/18683239/28020194-5f487cd0-657b-11e7-9deb-bded7990c291.png)

However, the change to Morecambe and Cumbria CCGs seems to have worked, with the new ones showing up:

![image](https://user-images.githubusercontent.com/18683239/28020245-8661eb4e-657b-11e7-94ff-2b34a2b4412f.png)

**Comment by richiecroker** [comment]
Also, all Manchester CCGs (new and old) appear in the CCG selection box:
![image](https://user-images.githubusercontent.com/18683239/28020318-d7709828-657b-11e7-839f-f3bc2d36981e.png)

as do the old Newcastle CCGs:

![image](https://user-images.githubusercontent.com/18683239/28020343-ebe41168-657b-11e7-9d88-11a85afeac01.png)

whereas the old Cumbria CCG doesn't:

![image](https://user-images.githubusercontent.com/18683239/28020374-03f111c0-657c-11e7-8c5e-aa91b43b07e7.png)


Ideally we'd be in a Cumbria rather than a Manchester situation.  Is this related to issue #479?

**Comment by sebbacon** [comment]
Irritatingly, the NHS boundaries file at https://www.england.nhs.uk/resources/ccg-maps/ does not include the new Manchester CCG, so I'll keep this issue open until we can get it updated.

**Comment by sebbacon** [comment]
This alternative data source has been proposed: http://geoportal.statistics.gov.uk/datasets/clinical-commissioning-groups-april-2017-full-clipped-boundaries-in-england-v4

But it doesn't import cleanly, so needs some work.

**Comment by sebbacon** [comment]
A major blocker to using that file is that it uses ONS ids for the CCGs rather than the NHS ones!  And I can't find a canonical source for mapping between the two.  Have a query with David Miller who thinks he might be able to chase down the right people.

**Comment by lmfrench** [comment]
https://www.england.nhs.uk/resources/ccg-maps/  now links to http://geoportal.statistics.gov.uk/datasets/clinical-commissioning-groups-april-2017-full-clipped-boundaries-in-england-v4 so not sure if we'll ever see an updated boundaries file

**Comment by lmfrench** [comment]
Looks like this could be used as a lookup: http://geoportal.statistics.gov.uk/datasets/lower-layer-super-output-area-2011-to-clinical-commissioning-group-to-local-authority-district-april-2017-lookup-in-england-version-4

**Comment by sebbacon** [comment]
OK. Writing code that does all that automatically is a massive pain the arse because they've also stopped supplying it in mapinfo format.

Given it rarely changes and each time it does, the format completely changes;l and that STPs are going to be the new CCGs over the next 12 months in any case;  I have decided not to bother automating the import. I've added some notes to the importer about manually massaging the data format, instead.
